internal/cmd/commands/database/init.go

```diff
@@ -275,6 +275,17 @@ func (c *InitCommand) Run(args []string) (retCode int) {
 		c.UI.Error(fmt.Errorf("Error setting up schema manager for locking: %w", err).Error())
 		return 1
 	}
+	// This is an advisory locks on the DB which is released when the db session ends.
```

Comment:
```suggestion
	// This is an advisory lock on the DB which is released when the DB session ends.
```
(I don't particularly care about DB/db but it ought to be consistent within the same comment line :-D )